src/Controls/tests/DeviceTests/Elements/CollectionView/CollectionViewTests.Windows.cs

```diff
@@ -101,7 +101,7 @@ void ValidateItemContainerStyle(CollectionView collectionView)
 			Assert.Equal(0d, minHeight);
 		}
 
-		[Fact(Skip = "FIX FOR .NET8")]
+		[Fact]
```

Comment:
> When I run this test, `OnScrollViewerFound` now gets called twice because `FindScrollViewer` wires up to loaded twice. Do we need to make sure `FindScrollViewer` doesn't wire up to `listView.Loaded` twice?

With the current architecture there's not much we can do about the Loaded wire-up (potentially) happening twice, but each one will unhook the event handler when it fires. And I've updated OnScrollViewerFound to unhook any previous ScrollChanged handlers (and to exit early if it's finding the same ScrollViewer). So even in the worst case of two Loaded handlers getting hooked up and firing, the worst thing that happens is an extra call to OnScrollViewerFound that exits immediately.